cpplint_unittest.py

```diff
@@ -5259,9 +5257,7 @@ def testBuildHeaderGuard(self):
             ],
             error_collector,
         )
-        for line in error_collector.ResultList():
-            if line.find("build/header_guard") != -1:
-                self.fail("Unexpected error: %s" % line)
+        assert not any("build/header_guard" in line for line in error_collector.ResultList())
```

Comment:
Although `ruff rule SIM110` recommends the use of `any()` because it is much easier to read and understand, it may make pytest results harder to understand because it may not show the text on those line(s) that failed.


Also, (I learned the hard way) that `any()` is actually slower than the long form that it replaces.

Reply:
Should I change them all to the early exit–less form `assert not "build/header_guard" in (line for line in error_collector.ResultList())`?